Fix `Unexpected token <` error

valgussev opened the pull request:
By default `skipWaiting` parameter is set to true, allowing newly
registered service worker to bypass the `waiting` state, so all out of
date cache entries from the previous service worker will be deleted.

Because of async dynamic module loading we don't want that to happen,
keeping a way of loading old chunks.

The downside of removing skipWaiting is that newly deployed
service worker code and caches won't become active until all existing
tabs that have an older version of that service worker are closed, but
that actually sounds we want exactly this behaviour. 

Related to
https://github.com/GoogleChromeLabs/sw-precache/issues/180#issuecomment-248386760

We can also provide users with a popup by click on which `skipWaiting()` will be called manually.

Any thoughts on this?

Commit:
Fix `Unexpected token <` error

By default skipWaiting parameter is set to true, allowing newly
registered service worker to bypass the `waiting` state, so all out of
date cache entries from the previous service worker will be deleted.

Because of async dynamic module loading we don't want that to happen,
keeping a way of loading old chunks.

The downside of removing skipWaiting is that newly deployed
service worker code and caches won't become active until all existing
tabs that have an older version of that service worker are closed, but
that actually sounds we want exactly this behaviour.

Related to
https://github.com/GoogleChromeLabs/sw-precache/issues/180#issuecomment-248386760

evancharlton reviewed (approved):
Seems reasonable to try it out.

halvard-cognite commented:
You were not able to reproduce this and test locally?

valgussev commented:
I reproduced locally, but I believe it could be the wrong one.
1. Create a new build `yarn build`
2. Served it with `firebase serve` ensuring we use exactly the same environment as in production
3. Opened the page, service worker has been installed and chunks have been cached
4. Removed one of the chunks from Cache storage in dev tools
5. Replaced build directory with another build from another brunch
6. Tried to navigate to a new chunk section -> Received an error

If to set `skipWaiting` to false, the problem will never occur, since to install the new service worker we need to reopen tab.
